1-exercises/A-array-find/exercise.js

```diff
@@ -3,7 +3,9 @@
   Using .find(), we'd like to find the first name which starts with A and is longer than 7 letters.
 */
 
-// write your code here
+function findLongNameThatStartsWithA(n) {
+  return n.length > 7 && n[0] === "A";
```

Comment:
Correct condition, should you use find function to find the names satisfies these conditions?

Reply:
line 22

Reply:
Ok, great!